cogs/moderation.py

```diff
@@ -16,12 +27,24 @@ def __init__(self, bot):
         with open('banned.txt', 'r', encoding="utf-8") as f:
             self.banned = f.read().splitlines()
 
+    def remove_invites(self, message):
+        isInvite = re.search(r'\b(?:https?:\/\/)?(?:www\.)?discord\.gg\/\w+\b', message)

```

Comment:
I forget regex - does this also account for case sensitivity? 

does this account for whitespace anywhere in the URL string? Perhaps we can remove all whitespaces from `message` if it makes things easier

Reply:
We shouldn't need to worry about case as we're passing `message.content.lower()`

It doesn't take whitespace into account. However I'm not sure that's something we need to worry about at this point.

I could go either way on it.

Reply:
> It doesn't take whitespace into account. However I'm not sure that's something we need to worry about at this point.

Yeah, I'm ok if we add it later when/if someone starts abusing the workaround